votrfront/js/PriebezneHodnoteniaPage.js

```diff
@@ -0,0 +1,84 @@
+
+import { ZapisnyListSelector } from './ZapisnyListSelector';
+import { CacheRequester, Loading } from './ajax';
+import { PageLayout, PageTitle } from './layout';
+import { Link } from './router';
+
+
+export var PriebezneHodnoteniaPageContent = React.createClass({
+  propTypes: {
+    query: React.PropTypes.object.isRequired
+  },
+    
+  renderContent() {
+    var cache = new CacheRequester();
+    var {zapisnyListKey} = this.props.query;
+    var [priebezneHodnotenia, message] = cache.get('get_priebezne_hodnotenia', zapisnyListKey) || [];
+
+    if (!priebezneHodnotenia) {
+      return <Loading requests={cache.missing} />;
+    }
+    
+    if (priebezneHodnotenia.length == 0) {
+      return <div>Žiadne priebežné hodnotenia.</div>;
+    }
+
+    return (
+      <div>
+      {priebezneHodnotenia.map((priebHod) =>
+        <div>
+          <h2><Link href={{ ...this.props.query, modal: 'detailPredmetu', modalPredmetKey: priebHod.predmet_key, modalAkademickyRok: priebHod.akademicky_rok }}>
+                {priebHod.nazov}</Link>
+          </h2>
+          <table className="table table-condensed table-bordered table-striped table-hover">
+            <thead>
+              <tr>
+                <th>Dôvod hodnotenia</th>
```

Comment:
Vsetky tie stlpce su jedna ku jednej z AISU, a "Zapocitavat" neviem presne co znamena, asi znamka alebo co.
Na to spojenie poctu bodov a maxima sa pozriem a uvidim co bude lepsie/krajsie.